omod/src/main/compass/sass/login.scss

```diff
@@ -8,4 +8,9 @@
       width: 405px;
     }
   }
-}
\ No newline at end of file
+}
+
+.location-error {
+	color: red;
```

Comment:
The style was not applied. Am guessing its because i have not actually included the css file inside the logn.gsp.
To include that css file, am suppose to use something like ui.includeCss("uicommon", "fileName.css"), correct? If yes which file should i replace with fileName.css. If no, how to include the generated css inside login.gsp.

I thought about this because login.css is included using ui.includeCss("referenceapplication", "login.css") so am guessing am suppose to do something similar but replace "referenceapplication" with "uicommons" but i don't know which generated css file should i replace "login.css" with. I don't have enough internet data now so i can't clone uicommons and build and look for the generated file that will hold the .field-error. I will probably do that tomorrow when i have bought some internet data. But if you know which file it is am suppose to include, you can just save me the trouble and lets finish with this ticket today. :)